source/tutorials/tutorial.md

```diff
@@ -76,27 +86,17 @@ So, the first thing you want to do is find the perfect spot to settle down. But,
+**Hint:** If you're not using the building tool and aren't able to place the Supply Camp/Ship, try it block by block: one to the right, left, closer or further from the area you cleared. The area cannot contain any holes or have any flowers, grass, ferns, seaweed, coral etc.
 
 ### Town Hall
 
- The protected area of your colony (once the [Town Hall](../../source/buildings/townhall) has been placed) will be a 8 chunks radius, measured from where you placed your Town Hall block the first time (Default config). Therefore, plan carefully where you want to place your Town Hall. 128 blocks (8 chunks x 8 chunks) in every direction will be your protected Town area including mountains, hills, lakes, oceans, caves, world generated structures, etc. from bedrock to the sky limit.
+ The protected area of your colony (once the [Town Hall](../../source/buildings/townhall) has been placed) will be a square 8 chunks in radius, measured from where you placed your Town Hall block the first time (Default config). Therefore, plan carefully where you want to place your Town Hall. 128 blocks (8 chunks x 8 chunks) in every direction will be your protected Town area including mountains, hills, lakes, oceans, caves, world generated structures, etc. from bedrock to sky limit.
 
 - **Note**: You can configure your own town area radius in the configuration options of the mod or the the minecolonies-common.toml file inside the ```../minecraft/config/``` folder of your own world.
 
-Due to the protected area of each colony, you have to scout carefully your surroundings to make sure you are clear of any other colonies nearby preventing you from placing your Town Hall or limiting your colony area in that direction.
-
-If there is another colony too close to your current position, you won't be able to place a Town Hall.
+Due to the protected area of each colony, you have to carefully scout your surrounding to make sure you are clear of any other colonies nearby preventing you from placing your Town Hall or limiting your colony area in that direction (in multiplayer, etc.). If there is another colony too close to your current position, you won't be able to place a Town Hall.
 
-If you want to know whether there is a colony too close to your current position, we've included some information in the debug screen (displayed when you press F3) which contains a real-time tracker showing you the distance to the next colony, the colony you're currently in, or if there is no colony close at all. Plus, if there is a colony close to your current position, it will also show you the required distance away to place your Town Hall.
+Location info of the next closest colony exits in the debug screen (displayed when you press F3). If there is a colony close to your current position, it will also show you the required distance away to place your Town Hall.
```

Comment:
```suggestion
Location info of the closest colony exits in the debug screen (displayed when you press F3). If there is a colony close to your current position, it will also show you the required distance away to place your Town Hall.
```